### DISCUSSION TITLE
FO - Product - If a pack contains customizable product I cannot customize it when I add pack to cart

### DISCUSSION THREAD

**Discussion by DianaButkute** [PrestaShop/PrestaShop#38336] [Ideas]
#### Describe the bug

Creating the product pack and adding the product which has customization like text (as example I am selling the t-shirt and client can write custom text on the t-shirt). After the product select into the BO I do not see the inputs where can I insert my custom values. 

#### Expected behavior

The customization fields (text or files) should appear into the BO when creating pack and selected product which has it.

#### Steps to Reproduce

- create 2 customizable products in BO
- create a pack of products containing these 2 products
- go to FO, can't add customization in products of the pack (if custom is required)

**Screenshots**

![image](https://user-images.githubusercontent.com/51438018/82901129-2be30400-9f66-11ea-8208-bdff2dc23ab9.png)

#### Additional information

* PrestaShop version: 1.7.7.0
* PHP version: 7.1

[a comment by an automated account is not shown]

**Comment by marionf** [comment]
Hello @DianaButkute 

> I do not see the inputs where can I insert my custom values.
> And you will see that the product are added but no text fields or file inputs apperars for this product.

Are you talking of the back-office or front-office ?
In BO, it's normal as you should configure the customizable field in the customizable product and not in the pack

**Comment by DianaButkute** [comment]
Hello @marionf 
 
> Are you talking of the back-office or front-office ?
> In BO, it's normal as you should configure the customizable field in the customizable product and not in the pack

I am toking about BO - back-office. Yes it is normal, but if I want to create pack from customizable product. It means that I want to have pack with two products, one would be the women t-shirt (where you can write whatever you want on it) and the men t-shirt (where you can write whatever you want on it). How to solve this problem ? Because when I creating pack I do not see any labels or file including settings and also into the FO I do not see the place where can I write customizable informationa bout my those two products, which are included into the pack.

**Comment by florine2623** [comment]
Hi @DianaButkute ,
I did manage to reproduce your issue. This is a bug happening in both PS 1.7.6.5 and PS 1.7.7.0.
I’ll add this to the debug roadmap so that it’s fixed. If you have already fixed it on your end or if you think you can do it, please do send us a pull request!

Steps to reproduce :
- create 2 customizable products in BO
- create a pack of products containing these 2 products
- go to FO, can't add customization in products of the pack (if custom is required)

Thanks!
